recipes-bsp/sgx/sgx-dcap_1.9.bb

```diff
@@ -0,0 +1,105 @@
+# Packages supported
+# sgx-dcap : SGX Data Center Attestation Package.
+
+### common ###
+
+SUMMARY = "Intel(R) SGX Data Center Attestation Primitives (DCAP) driver"
+LICENSE = "BSD & Intel-Redistributable-Binaries"
+LIC_FILES_CHKSUM = "file://License.txt;md5=66da782d3c2c41b9669340fb2704ac59"
+
+# Packages required to build.
+DEPENDS += "curl sgx-sdk-cross"
+
+# Source locations
+S = "${WORKDIR}/git"
+D_prebuilt_dcap = "${S}/QuoteGeneration"
+D_sgxssl="${S}/QuoteVerification/sgxssl"
+D_openssl="${S}/QuoteVerification/sgxssl/openssl_source"
+
+SRC_URI  = "git://git@github.com/intel/SGXDataCenterAttestationPrimitives.git"
+SRCREV = "98976322e8b58e23256355f5cf90b9e30e37d8c1"
+
+### configure prebuilt ###
+SRC_URI += "https://download.01.org/intel-sgx/sgx-dcap/1.9/linux/prebuilt_dcap_1.9.tar.gz;name=prebuilt_dcap;subdir=${D_prebuilt_dcap}"
+SRC_URI[prebuilt_dcap.sha256sum] = "8cd0249ee49dbfd589b257cd0fa14d374d01b6c210ca7d0a14e618a48bbdb82b"
+
+configure_prebuilt () {
+    :
```

Comment:
what does this override function doing here ? at least add a comment here whatever we trying to do here.

Reply:
removed this part.